nservicebus/messaging/routing-extensibility_distribution_core_[6,6.2).partial.md

```diff
@@ -0,0 +1,7 @@
+include: routing-extensibility.distribution
```

Comment:
partials must match the version numbers of the snippets dirs. so unless u want to add a 6.2 here https://github.com/Particular/docs.particular.net/tree/master/Snippets/Core this partial will be ignored